### PR TITLE
(WIP) January 2018 Multiple Integrations

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description
The goal of this PR is to implement the changes and functionality for multiple pull requests, to include the following:

- Python 3 upgrade #70
- Standard Bounties #140
- Github Integration / Express Interest #156

##### Checklist
- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Refers/Fixes
Refs: #70, #140, #156

### CODE REVIEW

**Comment by mkosowsk** [comment]
Code coverage seems a bit low here, is that accurate? Might be interesting to create some issues in the near future to get these numbers up, could be some nice beginner issues 👍

**Comment by speizerj** [comment]
@mbeacom everything working ok with Python 3 here? There's a lot of additional code and I'm happy to sort out any issues that have come up as a result. Should be able to pull the branch down and test out later this week if it's still necessary by then.

**Comment by mbeacom** [comment]
@speizerj Thank you for following up! Yeah, there is quite a bit going into this PR.  Normally we wouldn't squash these into a large pull request, but we need to test everything for all three of these merged PRs anyway. Everything with Python 3 / Django 2.0 appears to be functioning properly.

Thanks again for the contribution!

File: app/dashboard/models.py [hunk 1]
```diff
@@ -270,6 +270,19 @@ def desc(self):
...
+        """Determine if the Bounty is legacy based on sunset date.
+
+        Todo:
+            * Remove this method following legacy bounty sunsetting.
+
+        Returns:
+            bool: Whether or not the Bounty is using the legacy contract.
+
+        """
+        return (self.created_on < settings.LEGACY_CONTRACT_SUNSET)
```
Comment (owocki) [inline app/dashboard/models.py]: `return self.web3_type != 'bounties_network'` will accomplish this too

File: app/dashboard/models.py [hunk 2]
```diff
@@ -74,27 +99,38 @@ class Meta:
     bounty_owner_address = models.CharField(max_length=50)
     bounty_owner_email = models.CharField(max_length=255, null=True)
     bounty_owner_github_username = models.CharField(max_length=255, null=True)
+    bounty_owner_name = models.CharField(max_length=255, null=True)
     claimeee_address = models.CharField(max_length=50)
-    claimee_email = models.CharField(max_length=255, null=True)
-    claimee_github_username = models.CharField(max_length=255, null=True)
-    is_open = models.BooleanField(help_text='Whether the bounty has been filled')
+    claimee_email = models.CharField(max_length=255, null=True, blank=True)
```
Comment (mbeacom) [inline app/dashboard/models.py]: @owocki @jasonrhaas  Can we adjust all of these text/char fields to: `blank=True` vs `null=True`?  Per https://docs.djangoproject.com/en/2.0/ref/models/fields/#null

Comment (jasonrhaas) [reply app/dashboard/models.py]: @mbeacom Sounds good to me.

Comment (owocki) [reply app/dashboard/models.py]: 👍

File: app/assets/v2/js/abi.js [hunk 3]
```diff
@@ -12,10 +12,10 @@ var bounty_address = function (){
             return '0x2af47a65da8cd66729b4209c22017d6a5c2d2400';
         break;
         case "ropsten":
-            throw "not supported";
+            throw "this network is not supported in bounty_address() for gitcoin";
```
Comment (mbeacom) [inline app/assets/v2/js/abi.js]: 🙌

**Comment by jasonrhaas** [comment]
🍰

**Comment by owocki** [comment]
bees

**Pull request merged**